### DISCUSSION TITLE
Should I continue using the Godot 3 build or move to the current Godot 4 build? (sorry if this question is bothersome)

### DISCUSSION THREAD

**Discussion by The-HandyDandyman** [Techokami/SonicWorldsNext#85] [Q&A]
I'm not sure if this is a question that's suitable to ask here, but I'm struggling to figure out what to do elsewhere and I keep going back and forth since I'm incredibly tied between what I want to do, so I figured I should ask this here, as somebody who's interested in the new features of Sonic Worlds Next's Godot 4 build. 
I've been using the Godot 3 builds before the update came out in order to work on a personal project, and I've gotten a decent way through: I've made a custom moveset, I've made lots of sprites and a background, utilizing the chunk system and slowly working on taking the info I learned from the CD time travel question to flesh out what I currently have. I know Godot 4 has new features to utilize and I wanna give this framework's new features a shot, but at the same time I'm conflicted due to the fact I've utilized the chunk system which, unless I'm overlooking things, is currently kind of busted in these newer builds, Godot 4's editor feels slightly more sluggish to me, and I don't want to redo the bits of work I've already done if it's going to cause issues in the long run.

Apologies for the rambling-point is, would it be better for the long run to continue my work in Godot 3.5 or try to advance forward to the newest builds for Godot 4? If anybody could give me advice on how I should go about things, that would be greatly appreciated. Thank you, have a good day!

**Comment by DimensionWarped** [comment]
That's ultimately your call of course. We aren't backporting things to the GD3.5 version, but you always can do that yourself if you see something new that you want.

**Reply by The-HandyDandyman** [reply]
Thank you for the response! Funnily enough I was actually considering that, I might give it a shot myself to see if that'll work out (From my understanding, the coding side has barely changed so at most I only need to worry about the functionality, tile and physics side of things, although do correct me if I'm wrong on that.)

**Comment by Renhoex** [comment]
I'm in a similar boat, my recommendation is just stick with 3.5 unless there's features you absolutely need in godot 4

**Reply by The-HandyDandyman** [reply]
Thanks for responding!! I was frankly leaning towards sticking with 3.5 anyhow, I sort of realized how long it'd take to get animations back and get certain things working as they should. I just wanted to double check and make sure that'd be a good course of action.

**Reply by DimensionWarped** [reply]
If you are good with a text editor you could just isolate your player scenes, convert them, and then copy that over to the new project pretty quickly without having to do anything tedious like remaking animations.
